# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxWindowsPlatform.h
# Commit: ca2e6339f35f
# Full Hash: ca2e6339f35f38cc26a203f29c8dbfeb38f96afc
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-03-14 09:41:39
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r aa4e010199c6 -r ca2e6339f35f gfx/thebes/gfxWindowsPlatform.h
--- a/gfx/thebes/gfxWindowsPlatform.h	Mon Mar 13 21:04:10 2023 +0000
+++ b/gfx/thebes/gfxWindowsPlatform.h	Mon Mar 13 21:04:10 2023 +0000
@@ -200,7 +200,13 @@
 
  protected:
   bool AccelerateLayersByDefault() override { return true; }
+
   nsTArray<uint8_t> GetPlatformCMSOutputProfileData() override;
+
+ public:
+  static nsTArray<uint8_t> GetPlatformCMSOutputProfileData_Impl();
+
+ protected:
   void GetPlatformDisplayInfo(mozilla::widget::InfoObject& aObj) override;
 
   void ImportGPUDeviceData(const mozilla::gfx::GPUDeviceData& aData) override;